packages/firebase_database/test/firebase_database_test.dart

```diff
@@ -142,6 +147,29 @@ void main() {
           ]),
         );
       });
+
+      test('runTransaction', () async {
```

Comment:
I think we could make this test more comprehensive by simulating the doTransaction message from the native side. See https://github.com/flutter/plugins/blob/master/packages/firebase_database/test/firebase_database_test.dart#L205 for an example of how to do this.

Reply:
Add simulation of the doTransaction message from the native side to mock method call handler.